### ISSUE TITLE
Missing link to package at https://www.rformassspectrometry.org/contributions/

### ISSUE THREAD

**Issue by tobiasko** [fgcz/MsBackendRawFileReader#18]
https://www.rformassspectrometry.org

does not mention this package at all. There is also no link to your name in the contribution section @cpanse. This should really change!

**Comment by cpanse** [comment]
I guess , a pull request would solve it

**Comment by cpanse** [comment]
https://github.com/rformassspectrometry/RforMassSpectrometry.org/pull/16

**Comment by tobiasko** [comment]
The task is completed once your name shows up on the homepage! ;-)

**Comment by jorainer** [comment]
Note: we put *only* developers and packages on the RforMassSpectrometry homepage that are on the RforMassSpectrometry github repo. We maintain and are responsible for all these packages for which we also have full control. Contributors listed on the "Contributors" page have provided code to these packages.

That being said, I think it would be maybe good to have a separate section/page with *outside* contributors and packages that are not part of RforMassSpectrometry, but that implement/extend/support that. IMHO that would be even better than adding your package to the RforMassSpec repo. Any thoughts on that?

**Comment by tobiasko** [comment]
I also thing that adding the package to the RforMassSpec repo would be the easier solution. @cpanse?

**Comment by cpanse** [comment]
@jorainer great; fine with us. let me handle the current pull request and than we can transfer it. What do I need to do?

**Comment by jorainer** [comment]
IMHO there is no need to transfer the package to RforMassSpectrometry - I think it's perfectly fine to have it in your repo. There is no need that all packages that extend e.g. `Spectra` (or `MsBackend`) need to be at RforMassSpec.

Ideal solution for me: 
- you maintain your package and keep it at fgcz
- you make a PR to the `Spectra` vignette adding your backend (there is a section listing additional backends).
- ideally, your package should be added to Bioconductor, so it would be hosted there and it would be easily findable by users.

Transferring the package to RforMassSpectrometry would also mean that we would need to take over (at least part) of the maintainance duties - and that's something I simply can not do. There are also other packages around providing `MsBackend` backends, not hosted or maintained by us (e.g. [MsBackendMS](https://github.com/sneumann/MsBackendMS) or [MsBackendSirius](https://github.com/michaelwitting/MsBackendSirius))

**Comment by tobiasko** [comment]
Has the wind just turned 180 degrees? 😄

**Comment by tobiasko** [comment]
IMHO: It would be good to get listed [here](https://rformassspectrometry.github.io/Spectra/articles/Spectra.html#backends), but I fear the overall effect on package findability will be limited. A direct link on the homepage to all currently available backends could really help to guide potential users that are looking for a specific support. Maybe ordered by data formats?

**Comment by jorainer** [comment]
> Has the wind just turned 180 degrees?

no? we never proposed to move the package to R4MS - and as explained, I don't think that's necessary (and actually good). It's more efficient to have independently maintained packages.

> IMHO: It would be good to get listed here

yes, that's exactly what I meant with *in the Spectra vignette* above. Please make a PR to that vignette adding a link to your package. And yes, we have to find a better way to list backends (from R4MS and external ones). Whether the R4MS homepage is the correct place I don't know (also because we're not regularly updating that). Maybe the best place would be the `Spectra` package/repo, since the backend concept is only important for that package.

**Comment by tobiasko** [comment]
ahhh, ok! Sorry. I guess my eyes skipped the `than` in your comment. I basically picked up: it would be even better adding your package to the RforMassSpec repo.

**Comment by tobiasko** [comment]
ok. So our action item is `Please make a PR to that vignette adding a link to your package.` @cpanse Would you do that?

**Comment by jorainer** [comment]
Exactly. In this file there is a section *Backends* (to the end of the file) that lists already some other backends. Please add a link to yours too, ideally also with a short description what it provides and which data it can handle.

**Comment by cpanse** [comment]
@jorainer @lgatto @tobiasko https://github.com/rformassspectrometry/Spectra/pull/273/commits/e1dcac400d54ac673e4010723adfcddb1ffc56ad I hope you like it. Otherwise, please feel free to change it.

**Comment by jorainer** [comment]
Thanks! Better you change it - I asked for some changes in your PR :)

**Comment by cpanse** [comment]
@tobiasko https://bioconductor.org/packages/devel/bioc/vignettes/Spectra/inst/doc/Spectra.html#4_Backends
